README.md

```diff
@@ -185,12 +214,23 @@ c2rust transpile --binary myprog path/to/compile_commands.json
 
 Where `--binary myprog` tells the transpiler to use
 the `main` function from `myprog.rs` as the entry point for a binary.
+This can be repeated multiple times for multiple binaries.
 
 The translated Rust files will not depend directly on each other like
 normal Rust modules.
 They will export and import functions through the C API.
 These modules can be compiled together into a single static Rust library or binary.
 
+You can run with `--reorganize-definitions` (which invokes `c2rust-refactor`),
+which should deduplicate definitions and directly import them
+with `use`s instead of through the C API.
+
+The refactorer can also be run on its own to run other refactoring passes:
+
+```sh
+c2rust refactor $transform --cargo
```

Comment:
```suggestion
c2rust refactor --cargo $transform
```
Options usually come first, I'd change this to make it more like what people are used to.

Reply:
I had it this way because that's the order in https://github.com/immunant/c2rust-testsuite/blob/7345f356cbf4b5f1851f78cb6719aa212e398716/tests/templates.py#L70-L73, but this is more common for options in general, so I'll switch.